core/modules/replica/IngestHttpSvcMod.cc

```diff
@@ -139,7 +140,7 @@ json IngestHttpSvcMod::executeImpl(string const& subModuleName) {
     string const table = body().required<string>("table");
     string const columnSeparatorStr = body().required<string>("column_separator");
     if (columnSeparatorStr.empty() || columnSeparatorStr.size() != 1) {
-        throw invalid_argument(string(__func__) + " column separator must be a single character string");
+        retryAllowedError(__func__, "column separator must be a single character string");
```

Comment:
This also does not feel like retrying would work.

Reply:
Agreed. See my previous comment.